cvat/apps/engine/task.py

```diff
@@ -294,6 +297,57 @@ def _download_data(urls, upload_dir):
 def _get_manifest_frame_indexer(start_frame=0, frame_step=1):
     return lambda frame_id: start_frame + frame_id * frame_step
 
+class InvalidManifestError(Exception):
+    """
+    Indicates an invalid manifest file uploaded
+    """
+
+def _read_dataset_manifest(path: str) -> ImageManifestManager:
+    """
+    Reads an upload manifest file
+    """
+
+    if is_dataset_manifest(path):
+        return ImageManifestManager(path)
+    elif not is_manifest(path):
+        raise InvalidManifestError(
+            "Can't recognize a manifest file in "
+            "the uploaded file '{}'".format(os.path.basename(path))
+        )
+    return None
```

Comment:
The annotated result type is `ImageManifestManager`, and looking at this function's usage, a `None` value is not expected, so `None` shouldn't be returned.